pnpm-lock.yaml

```diff
@@ -132,43 +132,43 @@ importers:
   packages/e2e-tests/_test_dependencies/svelte-api-only:
     dependencies:
       svelte:
-        specifier: ^5.0.0-next.144
-        version: 5.0.0-next.144
+        specifier: ^5.0.0-next.151
+        version: 5.0.0-next.151
 
   packages/e2e-tests/_test_dependencies/svelte-exports-simple:
     dependencies:
       e2e-test-dep-cjs-only:
         specifier: file:../cjs-only
-        version: file:packages/e2e-tests/_test_dependencies/cjs-only
+        version: link:../cjs-only
```

Comment:
this change seems to result in the cjs-only package to not be transformed to esm compat anymore. :/

Reply:
`dedupe-injected-deps=false` fixes it